charts/ccsm-helm/templates/curator-configmap.yaml

```diff
@@ -0,0 +1,128 @@
+        options:
+          timeout_override:
+          continue_if_exception: False
+          disable_action: False
+          ignore_empty_list: True
+        filters:
+          - filtertype: pattern
+            kind: prefix
+            value: zeebe-
+          - filtertype: age
+            source: name
+            direction: older
+            timestring: '%Y-%m-%d'
+            unit: days
+            unit_count: {{ .Values.retentionPolicy.zeebeIndexTTL }}
+            field:
+            stats_result:
+            epoch:
+            exclude: False
+      # delete operate- indicies
+      2:
+        action: delete_indices
+        description: "Clean up ES by deleting old Operate indices"
+        options:
+          timeout_override:
+          continue_if_exception: False
+          disable_action: False
+          ignore_empty_list: True
+        filters:
+          - filtertype: pattern
+            kind: prefix
+            value: operate-
+          - filtertype: age
+            source: name
+            direction: older
+            timestring: '%Y-%m-%d'
+            unit: days
+            unit_count: {{ .Values.retentionPolicy.operateIndexTTL }}
+            field:
+            stats_result:
+            epoch:
+            exclude: False
+      # delete tasklist- indicies
+      3:
+        action: delete_indices
+        description: "Clean up ES by deleting old Tasklist indices"
+        options:
+          timeout_override:
+          continue_if_exception: False
+          disable_action: False
+          ignore_empty_list: True
+        filters:
+          - filtertype: pattern
+            kind: prefix
+            value: tasklist-
+          - filtertype: age
+            source: name
+            direction: older
+            timestring: '%Y-%m-%d'
+            unit: days
+            unit_count: {{ .Values.retentionPolicy.tasklistIndexTTL }}
+            field:
+            stats_result:
+            epoch:
+            exclude: False
+      # or delete indicies which exceed the total size of 10 gig
+    {{- if .Values.retentionPolicy.zeebeIndexMaxSize }}
+      4:
+        action: delete_indices
+        description: "Clean up ES by deleting too big Zeebe indices"
+        options:
+          timeout_override:
+          continue_if_exception: False
+          disable_action: False
+          ignore_empty_list: True
+        filters:
+          - filtertype: pattern
+            kind: prefix
+            value: zeebe-
+          - filtertype: space
+            disk_space: {{ .Values.retentionPolicy.zeebeIndexMaxSize }}
+            source: name
+            timestring: '%Y-%m-%d'
+    {{- end }}
+  config.yml: |-
+    ---
+    # Remember, leave a key empty if there is no value.  None will be a string,
+    # not a Python "NoneType"
+    client:
+      hosts:
+        - elasticsearch-master-headless
+      port: 9200
+      url_prefix:
+      use_ssl: False
```

Comment:
:thought_balloon: I imagine eventually people will request this to allow for fully secured clusters